airbyte-integrations/connectors/source-tidb/src/main/java/io/airbyte/integrations/source/tidb/TiDBSourceOperations.java

```diff
@@ -4,6 +4,33 @@
 
 package io.airbyte.integrations.source.tidb;
 
+import static com.mysql.cj.MysqlType.BIGINT;
```

Comment:
why not use `import import static com.mysql.cj.MysqlType.*` here as you are also using `import static io.airbyte.db.jdbc.JdbcConstants.*;` right below?